playground/TestShop/TestShop.AppHost/Properties/launchSettings.json

```diff
@@ -26,7 +27,8 @@
         "ASPIRE_DASHBOARD_OTLP_HTTP_ENDPOINT_URL": "http://localhost:16032",
         "ASPIRE_RESOURCE_SERVICE_ENDPOINT_URL": "http://localhost:17031",
         "ASPIRE_SHOW_DASHBOARD_RESOURCES": "true",
-        "ASPIRE_ALLOW_UNSECURED_TRANSPORT": "true"
+        "ASPIRE_ALLOW_UNSECURED_TRANSPORT": "true",
+        "Dashboard__ReverseProxy__ForwardHeaders": "true"
```

Comment:
This is probably answered in later code, but how does this env var to the host make its way to the dashboard?